index.scss

```
@@ -0,0 +1,3 @@
+@import './src/scss/global.scss';
+@import './src/scss/colors.scss';
+@import './src/components/Button/Button.scss';
```

For neatness I would suggest adding a separation between the global styles and the upcoming component styles

```suggestion
// Global styles
@import './src/scss/global.scss';
@import './src/scss/colors.scss';

// Component styles
@import './src/components/Button/Button.scss';
```